index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  v
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="bg">
+    <h1 id="title">
+      freeCodeCamp Survey Form
+    </h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+
+    <form id="survey-form">
+      <div class="form-group">
+        <label for="name" id="name-label">
+          Name
+        </label>
+        <input type="text" required id="name" placeholder="Enter your name">
+      </div>
+
+      <div class="form-group">
+        <label for="email" id="email-label">
+          Email
+        </label>
+        <input type="email" required id="email" placeholder="Enter your email">
+      </div>
+
+      <div class="form-group">
+        <label for="number" id="number-label">Age(optional)</label>
+        <input type="number" min="3" max="20" id="number" placeholder="Age">
+      </div>
+      <div class="form-group">
+        <label for="dropdown" id="dropdown-label">Which option best describes your current role?</label>
+        <select name="dropdown" id="dropdown">
+          <option value="Select an option" disabled>Select an option</option>
+          <option value="Challanges">Challanges</option>
+          <option value="Projects">Projects</option>
+          <option value="Comunity">Comunity</option>
+          <option value="Open source">Open source</option>
+        </select>
+      </div>
+
+      <div class="form-group">
+        <p id="dropdown-label">Would you recommend freeCodeCamp to a friend?</p>
+        <label class="flex-block">
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Definitely
+        </label>
+        <label class="flex-block">
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Maybe
+        </label>
+        <label class="flex-block">
+          <input name="user-recommend" value="definitely" type="radio" class="input-radio">
+          Not sure
```

Comment:
Add a semantic HTML tag like <section>

Reply:
[ Required ] The purpose of giving a code review is to help improve the project so it's important that you explain to the student the semantic tag that he should use and the benefit that semantic tag will have on his code base.

Please consider editing this review while factoring in my suggestions.

Reply:
Describing the contents of the page more semantically can have many benefits, such as:

Making code easier for people to navigate when updating features or fixing bugs.
Giving your page a higher Search Engine Optimization (SEO) score, which makes your content more visible on search engines.
Allowing users with special needs to access your website using alternative means such as screen readers, which are devices that read the contents of a webpage aloud to users who have visual impairments.